client/.stylelintrc

```diff
@@ -17,24 +44,63 @@
         ".*\\.lt-ie.*"
       ]
     },
+    "sh-waqar/declaration-use-variable": [["/color/", "font-size"]],
+    "declaration-bang-space-before": "always",
+    "declaration-bang-space-after": "never",
+    "declaration-property-value-blacklist": {
+      "/^border/": ["none"]
+    },
+    "color-named": "never",
+    "declaration-block-no-duplicate-properties": true,
+    "rule-nested-empty-line-before": [
+      "always", {
+        "except": [
+          "first-nested"
+        ],
+        "ignore": [
+          "after-comment"
+        ]
+      }
+    ],
+    "rule-non-nested-empty-line-before": [
+      "always", {
+        "ignore": [
+          "after-comment"
+        ]
+      }
+    ],
+    "block-no-empty": true,
+    "no-eol-whitespace": true,
+    "no-missing-end-of-source-newline": true,
     "color-no-invalid-hex": true,
     "color-hex-case": "lower",
+    "selector-no-id": true,
+    "declaration-no-important": true,
+    "number-leading-zero": "always",
+    "no-duplicate-selectors": true,
+    "max-nesting-depth": 3,
+    "selector-no-qualifying-type": true,
+    "declaration-block-semicolon-newline-after": "always-multi-line",
+    "selector-list-comma-newline-after": "always",
+    "function-comma-space-after": "always-single-line",
     "declaration-colon-space-after": "always",
+    "declaration-colon-space-before": "never",
+    "block-opening-brace-space-before": "always",
+    "function-parentheses-space-inside": "never",
+    "string-quotes": "single",
+    "declaration-block-trailing-semicolon": "always",
+    "number-no-trailing-zeros": true,
+    "function-url-quotes": "always",
+    "property-no-vendor-prefix": true,
+    "selector-no-vendor-prefix": true,
+    "media-feature-name-no-vendor-prefix": true,
+    "at-rule-no-vendor-prefix": true,
+    "value-no-vendor-prefix": true,
     "max-empty-lines": 1,
     "indentation": 2,
```

Comment:
Cheekily didn't change this to `4` – was going to see if I could get away with it.